README.md

```diff
@@ -18,22 +24,22 @@ This project automates the setup of a development environment for working on Rub
 
 Building the virtual machine is this easy:
 
-    host $ git clone https://github.com/rails/rails-dev-box.git
-    host $ cd rails-dev-box
+    host $ git clone https://github.com/aidanharan/activerecord-sqlserver-adapter-dev-box.git
+    host $ cd activerecord-sqlserver-adapter-dev-box
     host $ vagrant up
 
 That's it.
 
 After the installation has finished, you can access the virtual machine with
 
     host $ vagrant ssh
-    Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
+    Welcome to Ubuntu 18.04.4 LTS (GNU/Linux 4.15.0-96-generic x86_64)
```

Comment:
I guess this should remain on 20.04, which is what we have on Vagrantfile.

Reply:
SQL Server on Ubuntu doesn't support 20.04, see https://docs.microsoft.com/en-us/sql/linux/quickstart-install-connect-ubuntu

Reply:
SQL Server on Ubuntu doesn't support 20.04, see https://docs.microsoft.com/en-us/sql/linux/quickstart-install-connect-ubuntu

Reply:
It does https://packages.microsoft.com/config/ubuntu/20.04/

Reply:
Haven't tested Ubuntu 20.04 and Microsoft havent updated their installation instructions for it. I'll open a subsequent PR for Ubuntu 20.04.

Reply:
Looked at this again and SQL Server isn't supported on Ubuntu 20.04 yet. Looks like the https://packages.microsoft.com/config/ubuntu/20.04/ directory is just a placeholder for when they do have packages for Ubuntu 20.04. There is no equivalent https://packages.microsoft.com/config/ubuntu/18.04/mssql-server-2017.list in https://packages.microsoft.com/config/ubuntu/20.04/